File: buff/classes/354/pub/sum/go/sum.go
// Go sum program

package main

import . "fmt"

func sum(seq []int) (s int) {
    for _,v:=range seq {
        s+=v
    }
	return
}

func main() {
    seq:=[]int{5,6,1,8,3,7}
	Printf("%d\n",sum(seq))
}

File: buff/classes/354/pub/sum/java/Sum.java
// Java sum program

public class Sum {

    public static int sum(int[] seq) {
        int s = 0;
        for (int v : seq)
            s += v;
        return s;
    }

    public static void main(String[] args) {
        int[] seq = { 5, 6, 1, 8, 3, 7 };
        System.out.println(sum(seq));
    }

}
